md_core/update_med_views/views_all.py

```
@@ -0,0 +1,259 @@
+    ux = article_all_views('en', titles, 2024)
+    # ---
+    for t, tt in ux.items():
+        print(t, tt)
+    # ---
+    print(f"{len(ux)=:,}")
+
+
+def test():
+    # python3 core8/pwb.py update_med_views/views_all test
+    titles = load_lang_titles_from_dump("ar")
+    data = load_one_lang_views_all("ar", titles, "all")
```

Flake8 (7.2.0)

[error] local variable 'data' is assigned to but never used

(F841)